deployment/docker-compose.yml

```diff
@@ -48,6 +48,10 @@ uwsgi:
     - DJANGO_SETTINGS_MODULE=core.settings.prod_docker
     - VIRTUAL_HOST=geocontext.kartoza.com
     - VIRTUAL_PORT=8080
+    - RIVER_DATABASE_USER=geoserver
+    - RIVER_DATABASE_HOST=postgis.kartoza.com
```

Comment:
@andretheronsa rather place dummy values here. This is a public project and we wouldn’t want the world to hop into our DB